test/built-ins/Object/prototype/isPrototypeOf/context-is-in-prototype-chain-of-value.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2009 the Sputnik authors. All rights reserved.
```

Comment:
```suggestion
// Copyright (C) 2019 Alexey Shvayka. All rights reserved.
```

Reply:
Please see b3b31b4: this isn't a new test, but rather cleaned up and renamed [`S15.2.4.6_A1.js`](https://github.com/tc39/test262/blob/52fd2522b1b1c22d488382be5513c78e327aa853/test/built-ins/Object/prototype/isPrototypeOf/S15.2.4.6_A1.js), authored by Sputnik team.